### PR TITLE
Update readme.md for DeletionMarker remediation

### PR DESCRIPTION
**Pull request by mridul-ms**
## What is this change?
This is to add documentation for DeletionMarker Fixup added recently.

## What changed?
Updated readme.md for reglegacy fixups

### CODE REVIEW

File: fixups/RegLegacyFixups/readme.md [hunk 1]
```diff
@@ -108,6 +146,19 @@ Here is an example of using this fixup to address an application that contains a
...
+				},
+				{
+					"type": "DeletionMarker",
+					"hive": "HKCU",
+					"patterns": "^[Ss][Oo][Ff][Tt][Ww][Aa][Rr][Ee]\\\\Vendor\\\\.*"
+				},
+				{
+					"type": "DeletionMarker",
+					"hive": "HKLM",
+					"patterns": "^Software\\\\Vendor.*",
```
Comment (ravishroshanms) [inline fixups/RegLegacyFixups/readme.md:158]: it is not "patterns", but 'key'

File: fixups/RegLegacyFixups/readme.md [hunk 2]
```diff
@@ -108,6 +146,19 @@ Here is an example of using this fixup to address an application that contains a
 					"patterns": [
 						"^[Ss][Oo][Ff][Tt][Ww][Aa][Rr][Ee]\\\\Vendor\\\\.*"
 					],
+				},
+				{
+					"type": "DeletionMarker",
+					"hive": "HKCU",
+					"patterns": "^[Ss][Oo][Ff][Tt][Ww][Aa][Rr][Ee]\\\\Vendor\\\\.*"
```
Comment (ravishroshanms) [inline fixups/RegLegacyFixups/readme.md:153]: it is not "patterns", but 'key'

**Review by ravishroshanms — CHANGES_REQUESTED** [review]

File: fixups/RegLegacyFixups/readme.md (hunk 2 above)
Comment (mridul-ms) [reply fixups/RegLegacyFixups/readme.md]: Updated.

File: fixups/RegLegacyFixups/readme.md (hunk 1 above)
Comment (mridul-ms) [reply fixups/RegLegacyFixups/readme.md]: Updated.

**Review by ravishroshanms — APPROVED** [review]

File: fixups/RegLegacyFixups/readme.md [hunk 3]
```diff
@@ -65,17 +69,51 @@ The following Windows API calls are supported for this fixup type.
...
+> * RegOpenKeyEx
+> * RegOpenKeyTransacted
+
+### Configuration for DeletionMarker
+When the `type` is specified as `DeletionMarker`, the `remediation` element is an array of remediations with a structure shown here:
+| Tag | Purpose |
+| --- | ------- |
+| `hive` | Specifies the registry hive targeted. |
+| `key` | Contains a regex string. The key will be used against the path of the registry key being processed. |
+| `values` | Contains an array of regex strings. The value will be used against the name of the registry value being processed in the key specified. This is an optional field. Omitting this field would hide the key specified. |
```
Comment (npuvvada) [inline fixups/RegLegacyFixups/readme.md:104]: It looks values is optional from the example. Please mention it is optional and what is default value considered. Please refer to [this](https://github.com/microsoft/MSIX-PackageSupportFramework/tree/develop/PsfLauncher#json-schema)

**Review by npuvvada — APPROVED** [review]

**Pull request merged**